common/src/main/java/org/apache/rocketmq/common/MQVersion.java

```diff
@@ -16,9 +16,33 @@
  */
 package org.apache.rocketmq.common;
 
-public class MQVersion {
+import java.io.InputStream;
+import java.util.Properties;
+import org.apache.rocketmq.common.constant.LoggerName;
+import org.apache.rocketmq.logging.InternalLogger;
+import org.apache.rocketmq.logging.InternalLoggerFactory;
 
-    public static final int CURRENT_VERSION = Version.V4_5_2.ordinal();
+public class MQVersion {
+    private static final InternalLogger log = InternalLoggerFactory.getLogger(LoggerName.COMMON_LOGGER_NAME);
```

Comment:
Is that appropriate to use the same logger name ? I notice you use the warning semantic to print log, you could use the different log level and directory to logging the trivial info for problem diagnosis.